app/views/quotes/new.html.erb

```diff
@@ -59,14 +127,29 @@
          </div>
       </div>
    </div>
+   <div class="row">
+      <div class="col">
+         <div class="card text-center">
+            <h5 class="card-header font-weight-bold bg-primary text-white text-shadow">Having trouble?
+                  <button 
+                     type="button" 
+                     class="btn edit-button btn-sm" 
+                     data-toggle="modal" 
+                     data-target="#requestCallSubmitModal" 
+                     id="searchAddressButton">Request a call
+                  </button>
```

Comment:
Bad indentation

Reply:
Actually, this whole file is indented with 3 spaces instead of 2

Reply:
done